HOWTO.md

```diff
@@ -50,6 +56,15 @@ $ aws cloudformation package \
     --output-template-file packaged-template.yaml
 ```
 
+Using `aws-sam-cli`
+
+```bash
+$ sam package \
+    --template-file /path_to_template/template.yaml \
+    --s3-bucket bucket-name \
+    --output-template-file packaged-template.yaml
```

Comment:
This was just today being discussed in #830. The command for `aws cloudformation package` has the additional option of specifying --s3-prefix, which is recommended if you're going to be using S3 as an artifact repository.

Reply:
Sam most certainly reduces the amount of typing one has to do. sam init is fantastic. However some parts aren't necessarily very user friendly. For example sam package help 

```shell 
$ sam --version
SAM CLI, version 0.12.0
$ sam package help
Usage: sam package [OPTIONS] [ARGS]...

Error: Missing option "--s3-bucket".
```

Which is very different to the documentation brought up by aws cloudformation package help. But that takes over window, which is maybe (haven't tried) a problem for popen?